cmd/collector/main.go

```diff
@@ -95,3 +106,43 @@ func logOptions(loggingConfigPath *string) ([]zap.Option, error) {
 
 	return l.Options()
 }
+
+func checkManagerConfig(configPath *string) error {
+	if _, err := os.Stat(*configPath); err == nil {
```

Comment:
I think this if statement might need a restructure a bit. I'd like to avoid the `os.ErrInvalid` at the end. Let's sync up on this.